packages/manager/modules/pci/src/projects/project/data-processing/submit-job/submit-job.html

```diff
@@ -71,6 +70,37 @@
             >
             </pci-project-data-processing-submit-job-spark-config>
         </oui-step-form>
+
+        <oui-step-form
+            data-header="{{ ::'data_processing_submit_job_stepper_submit_title' | translate }}"
+            data-submit-text="{{ ::'data_processing_submit_job_button_label' | translate }}"
+            data-prevent-next="true"
+            data-on-submit="$ctrl.onSubmitJobHandler()"
+        >
+            <order-command
+                data-order-api-url="$ctrl.orderAPIUrl"
+                data-order-data="$ctrl.orderData"
+                data-help-link="$ctrl.submitJobGuideUrl"
+                data-auto-height="true"
+                data-user="$ctrl.user"
+            >
+                <span
+                    data-translate="data_processing_submit_job_command_description"
+                ></span>
+
+                <a
+                    class="oui-link_icon"
+                    data-ng-href="{{:: $ctrl.apiGuideUrl }}"
+                    target="_blank"
+                    rel="noopen"
+                >
+                    <span
+                        data-translate="data_processing_submit_job_command_api_link"
+                    ></span>
+                    <span class="oui-icon oui-icon-external-link"></span> </a
+                >.
```

Comment:
This dot can be part of the translation?

Reply:
Changed